plugins/cost-insights/src/utils/currency.ts

```diff
@@ -61,3 +61,12 @@ export const defaultCurrencies: Currency[] = [
     rate: 5.5,
   },
 ];
+
+export const createCurrency = (
```

Comment:
nit: This doesn't return a currency type. Maybe name this `createCurrencyFormat`?